### PR TITLE
SpeciesProperties: C, N, O, Cu

### PR DESCRIPTION
**Pull request by ax3l**
Add more ionic species as pre-defined particle species.

### CODE REVIEW

**Comment by MaxThevenet** [comment]
Great idea! Just a suggestion: should we try to link these properties with https://github.com/ECP-WarpX/WarpX/blob/development/Source/Utils/IonizationEnergiesTable.H? We could create a new table there, with the mass and charge of each species. This is generated with https://github.com/ECP-WarpX/WarpX/blob/development/Source/Utils/write_atomic_data_cpp.py, we could probably directly get the values of these properties from NIST too.

File: Source/Particles/SpeciesPhysicalProperties.H [hunk 1]
```diff
@@ -71,6 +87,14 @@ namespace species
...
         case PhysicalSpecies::hydrogen:
             return PhysConst::m_p;
+        case PhysicalSpecies::carbon:
+            return PhysConst::m_e * amrex::Real(22032.0);
+        case PhysicalSpecies::nitrogen:
+            return PhysConst::m_e * amrex::Real(25716.9);
+        case PhysicalSpecies::oxygen:
+            return PhysConst::m_p * amrex::Real(15.8834);
+        case PhysicalSpecies::copper:
+            return PhysConst::m_p * amrex::Real(63.0864);
```
Comment (lucafedeli88) [inline Source/Particles/SpeciesPhysicalProperties.H:97]: Is there a particular reason for expressing some masses as a function of  `m_p` and some others as a function of `m_e` ?

**Review by lucafedeli88 — COMMENTED** [review]
Thanks for this PR! I only left a minor comment...

Concerning possible improvements, I agree with @MaxThevenet  that it would be great to link these properties with those listed in https://github.com/ECP-WarpX/WarpX/blob/development/Source/Utils/IonizationEnergiesTable.H . However, I would not put these properties there. I would rather suggest to have a large table with more elements (at least those for which we have ionization tables) in `SpeciesPhysicalProperties.H` and to use these data in the ionization module. Having a script to scrape them NIST or some other database would really be great! I don't know if we want to do that in this PR or in another PR in the future.

File: Source/Particles/SpeciesPhysicalProperties.H (hunk 1 above)
Comment (ax3l) [reply Source/Particles/SpeciesPhysicalProperties.H:97]: No, I just copied the first ones [from a prior input](https://github.com/ComputationalRadiationPhysics/picongpu/blob/master/share/picongpu/examples/FoilLCT/include/picongpu/param/speciesDefinition.param) I used in that other PIC code on GPU that I worked with in the past :sweat_smile:

**Comment by ax3l** [comment]
Thank you both for the review!

I added the missing `Cu` ionization energies as in #260 accordingly :)

**Review by lucafedeli88 — APPROVED** [review]
Thanks for this PR!

**Pull request merged**